src/main/java/numberBaseballGame/domain/judgement/GameJudgement.java

```diff
@@ -0,0 +1,37 @@
+package numberBaseballGame.domain.judgement;
+
+public class GameJudgement {
+
+    public static final int MAX_COUNT = 3;
+
+    public static final int COUNT_ZERO = 0;
+    public static final int PLUS_AMOUNT = 1;
```

Comment:
상수간 개행을 둔 이유가 있을까요 👀
```suggestion
    public static final int MAX_COUNT = 3;
    public static final int COUNT_ZERO = 0;
    public static final int PLUS_AMOUNT = 1;
```